index_ghost_host.html

```diff
@@ -0,0 +1,373 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/ghost_host.css" />
+  <title>GhostHost</title>
+</head>
+
+<body class="wrapper">
+  <header class="wrapper">
+    <nav class="flexRow wrapper headerNav">
+      <a class="logo" href="markup-1.html">
```

Comment:
```suggestion
      <a class="logo" href="#">
```
this'll work the same, but without reload of the page

Reply:
And here, logo should be wrapped in h1